src/SldStyleParser.ts

```diff
@@ -169,18 +193,17 @@ class SldStyleParser implements StyleParser {
    * @return {ScaleDenominator} The GeoStyler-Style ScaleDenominator
    */
   getScaleDenominatorFromRule(sldRule: any): ScaleDenominator | undefined {
-    let min: number | undefined;
-    let max: number | undefined;
+    let scaleDenominator: ScaleDenominator = <ScaleDenominator> {};
     if (sldRule.MinScaleDenominator) {
-      min = parseFloat(sldRule.MinScaleDenominator[0]);
+      scaleDenominator.min = parseFloat(sldRule.MinScaleDenominator[0]);
     }
     if (sldRule.MaxScaleDenominator) {
-      max = parseFloat(sldRule.MaxScaleDenominator[0]);
```

Comment:
See comment above (float vs. int)

Reply:
SLD spec defines it as double so i'll stick with float.